### PR TITLE
missing step for add secret to build svc account

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@deads2k @liggitt ptal.

File: dev_guide/builds.adoc [hunk 1]
```diff
@@ -870,6 +870,16 @@ $ oc create -f secret.json
 ----
 ====
 
+. Add the `*secret*` to the builder service account:
++
+====
+
+----
+$ oc secrets add serviceaccount/builder Secrets/scmsecret
```
Comment (liggitt) [inline dev_guide/builds.adoc]: lowercase `secrets`?

**Comment by liggitt** [comment]
while you're in here, the whole base-64 discussion and secret JSON-building can be replaced with

```
oc secrets new scmsecret ssh-privatekey=$HOME/.ssh/id_rsa
```

**Comment by liggitt** [comment]
nits/improvements, LGTM

**Comment by bparees** [comment]
@liggitt updated.
@soltysh i was a bit confused by why there were two "oc create -f secret.json" steps in the doc previously, one of which referred to a Docker secret.  Can you please confirm that removing all that was the right thing to do here?  i think one of them was always redundant, and now neither are needed thanks to @liggitt 's suggested command.

**Comment by liggitt** [comment]
adding the docker secret would be needed to push images to or pull images from a private repo other than the built-in one

**Comment by bparees** [comment]
@liggitt yeah but that's covered in its own section already:
https://docs.openshift.org/latest/dev_guide/builds.html#using-docker-credentials-for-pushing-and-pulling-images

(a section which i should note uses the correct instructions :) )

**Comment by liggitt** [comment]
yeah, ok... I didn't notice the second docker secret bit... that does seem superfluous.

**Comment by bfallonf** [comment]
@bparees @liggitt I came across the second secret bit too when I first went through this doc. I questioned @soltysh and he told me it was necessary. You can see the conversation here: #443 (my question on line 313). Maybe double-check with him that it's all good to get rid of this from the docs?

**Comment by bparees** [comment]
@bfallonf thanks, that is why I tagged him on this initially :)  but i think in the PR you're pointing to it really was the two distinct sections (one for docker, one for source), i think that was different from what i removed.

**Comment by bparees** [comment]
@mfojtik can you pinch hit for @soltysh on reviewing this and in particular the question about why we were talking about docker secrets in the scm section?

**Comment by soltysh** [comment]
@bparees yeah, it's safe to remove it. I have no clue how it get there in he first place. Did you tested that the command @liggitt suggested works properly. I remember that SA works a bit differently from what we expect from SCM secrets,

**Comment by bparees** [comment]
I did not, but looking at what the instructions said to do i'm not seeing how it would be different...

what's "SA"?

**Comment by bparees** [comment]
@soltysh I just created a secret using the new way and compared its value to what the instructions would have had you do and it looks to be the same thing to me.

@adellape I think we're good to merge this.

**Comment by soltysh** [comment]
LGTM if so

**Comment by adellape** [comment]
> what's "SA"?

@bparees Service accounts?

**Comment by adellape** [comment]
LGTM

**Pull request merged**